src/main/java/org/embulk/util/csv/CsvTokenizer.java

```diff
@@ -135,6 +157,17 @@ public Builder noEscape() {
             return this;
         }
 
+        /**
+         * Returns the string to take place in a column value string tokenized from a quoted field containing an end-of-line.
```

Comment:
Let me check, just in case. ` a column value` = newline?

Reply:
No. This "column value" means literally a "column" value that may contain the configured "newline" converted from CR/LF.

Reply:
Thank you for your comment!